scipy/stats/_hypotests.py

```diff
@@ -1028,66 +1019,59 @@ def boschloo_exact(table, alternative="two-sided", n=32):
 
     References
     ----------
-    .. [1] Raised conditional level of significance for the 2 × 2‐table when testing
-           the equality of two probabilities. :doi:`10.1111/j.1467-9574.1970.tb00104.x`
+    .. [1] Raised conditional level of significance for the 2 x 2-table when
+           testing the equality of two probabilities.
+           :doi:`10.1111/j.1467-9574.1970.tb00104.x`
```

Comment:
We don't have a specific template for citations (yet), but should still be complete.
```suggestion
    .. [1] R.D. Boschloo. "Raised conditional level of significance for the
       2 x 2-table when testing the equality of two probabilities", 
       STATISTICA NEERLANDICA, 24(1), 1970
```